metadata.yaml

```diff
@@ -0,0 +1,19 @@
+name: training-operator
+display-name: Training Operator
+summary: Tools for distributed or non-distributed training on Kubernetes
+description: |
+  Training Operator provides Kubernetes custom resources that make it
+  easy to run distributed or non-distributed training jobs on Kubernetes.
+  This charm deploys the Operator configured for use with Kubeflow to
+  Kubernetes models in Juju.
+maintainers: [Juju Developers <juju@lists.ubuntu.com>]
```

Comment:
I'd prefer for this to include an actual name; this makes it look like the Juju team is maintaining this charm :)

Reply:
@knkski @DomFleischmann what do you guys think of this?

Reply:
There is currently a discussion about this maintainers field. I do agree that Juju Developers doesn't really make sense and I'm not sure if I'm even subscribed to this mailing list.

I personally feel like you can just add your own name and canonical email address there, that seems to be the direction charmhub is going anyways.

Reply:
In general, my preference is that there is an actual name associated with charms on Charmhub. That doesn't mean that the Kubeflow team can't be added as a contributor on the backend so you can all push/upload/release etc., but gives a more direct sense that it's 'owned'

Reply:
I updated it with my name and email address. Thanks.